bufferserver/src/main/java/com/datatorrent/bufferserver/internal/DataList.java

```diff
@@ -79,27 +93,31 @@ public void rewind(int baseSeconds, int windowId) throws IOException
     for (DataListIterator dli : iterators.values()) {
       dli.rewind(processingOffset);
```

Comment:
I think the rewind could be unnecessary for the current use case. I may have put it in there just for the completeness.

We can test it for its need and if deemed unnecessary, we can remove it. Alternately leave it there since as I explained, all the subscribers automatically get booted regardless due to redeployment.